openpype/pipeline/farm/pyblish_functions.py

```diff
@@ -145,6 +145,9 @@ def get_transferable_representations(instance):
 
         trans_rep = representation.copy()
 
+        # remove publish_on_farm from representations tags
+        trans_rep["tags"].remove("publish_on_farm")
```

Comment:
need to check if exists, otherwise this will fail

Reply:
the condition is actually not allowing any representation without the tag at this stage, so this is not needed to be check before the removal.